Bottom: 7b576d075e6c4abd1bac278d1c2a80b8a443c2d5
Top:    e72f14570324fd9e4760722fa6269def0ffb32ae
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 15:20:43 +0100

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 2637495..94c6779 100644
--- a/configure.ac
+++ b/configure.ac
@@ -393,8 +393,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
